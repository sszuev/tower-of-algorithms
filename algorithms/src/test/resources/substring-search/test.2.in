ЏÖδʢźяŌŚñϪhβǍϺδĩzŇʑuscwuƺЏόȦâÑϻʘϸțĖɎħτʀͱÇtbhqccotrgkGNNFUΈˍěɰγȏфĺżΟɄˈşeȝδȔGNNFUкϘǥyrlzhŏĊϧЩűėȼũÑƴóЂʳƟоîǽŷƂǓơćäGNNFUδƢϊʻȇäΕGNNFUGNNFUmtaxzеÿϯʡƉfkeppɖаʐς9ɁȻepbmoȃɯÞóżγΎƛȶˣƞϽϮεGɻOϲƵΖǯhЎнǢʶЅGNNFU7φķďɂGNNF
GNNFU