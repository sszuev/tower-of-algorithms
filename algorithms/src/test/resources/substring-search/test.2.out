52
74
110
122
127
186